kanidmd/idm/src/actors/v1_read.rs

```diff
@@ -44,7 +44,7 @@ use std::convert::TryFrom;
 
 pub struct QueryServerReadV1 {
     pub log_level: Option<u32>,
-    idms: Arc<IdmServer>,
+    pub idms: Arc<IdmServer>,
```

Comment:
Why make this pub?

Reply:
This was part of me chasing down weird paths, don't need it now - removed. 😄